middleware/webhooks.js

```diff
@@ -7,9 +7,16 @@ module.exports = function createWithWebhook({ secret, shopStore }) {
     const topic = request.get('X-Shopify-Topic');
     const shopDomain = request.get('X-Shopify-Shop-Domain');
 
+    // Shopify escapes forward slashes 
+    // + replaces '&' with \u0026 when the HMAC is built
```

Comment:
nitpick but can we use the word `and` instead of `+`. It took me a second to realized `+` was being used grammatically and not pseudo code

Reply:
second this